### PR TITLE
{tools}[gompi/2020a] YAXT v0.6.2

### PR DESCRIPTION
**Pull request by jedokaplan**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/y/YAXT/YAXT-0.6.2-gompi-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'YAXT'
+version = '0.6.2'
```
Comment (boegel) [inline easybuild/easyconfigs/y/YAXT/YAXT-0.6.2-gompi-2020a.eb:4]: @jedokaplan Why not update to the latest version (0.9.0), see https://www.dkrz.de/redmine/projects/yaxt/wiki/Downloads?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2626.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/3ac3e41aeb98e3a420c5aa0e19b74f68 for a full test report.

File: easybuild/easyconfigs/y/YAXT/YAXT-0.6.2-gompi-2020a.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/y/YAXT/YAXT-0.6.2-gompi-2020a.eb:4]: @jedokaplan ping question above?

**Comment by verdurin** [comment]
Hi @jedokaplan welcome to EasyBuild and thanks for your contribution. Do you have any updates on the suggestion from @boegel about updating to the latest version?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.6
See https://gist.github.com/15e083382b52dbf15cd33fbb0d912715 for a full test report.

**Comment by akesandgren** [comment]
Closing this PR, since it (only) uses deprecated toolchain versions,
see https://docs.easybuild.io/policies/toolchains .

We also have newer versions of YAXT available.

**Pull request closed without merging**